Source/JavaScriptCore/b3/B3PatchpointSpecial.cpp

```diff
@@ -162,13 +164,16 @@ MacroAssembler::Jump PatchpointSpecial::generate(Inst& inst, CCallHelpers& jit,
     offset += value->numChildren();
 
     StackmapGenerationParams params(value, reps, context);
+    jit.comment("Patchpoint body start, with arg value reps: ", listDump(reps));
+    jit.comment("generated by: ", value->sourceString());
```

Comment:
We shouldn’t be evaluating things as arguments to this method

Reply:
I guess it’s just a getter. But it feels like bad form

Reply:
Probably we should introduce `jitComment(jit, ...)` macro, which does not evaluate the arguments when comment is disabled.

Reply:
Comment is always inline, and does not consume its arguments when assertions are disabled

Reply:
Ooops, I am wrong